pkg/specgen/generate/namespaces.go

```diff
@@ -195,6 +195,12 @@ func namespaceOptions(ctx context.Context, s *specgen.SpecGenerator, rt *libpod.
 	// is not being created in a pod.
 	if s.IDMappings != nil && pod == nil {
 		toReturn = append(toReturn, libpod.WithIDMappings(*s.IDMappings))
+	} else if s.IDMappings != nil && pod != nil {
+		if pod.HasInfraContainer() {
+			if len(s.IDMappings.UIDMap) > 0 || len(s.IDMappings.GIDMap) > 0 {
+				return nil, errors.Wrapf(define.ErrInvalidArg, "Cannot specify a new uid/gid map when entering a pod with an infra container")
```

Comment:
```suggestion
				return nil, errors.Wrapf(define.ErrInvalidArg, "cannot specify a new uid/gid map when entering a pod with an infra container")
```
I think all error messages should be lowercase, @rhatdan correct?

Reply:
Yes